helm/object-storage-operator/templates/policy-exception.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   labels:
     {{- include "labels.common" . | nindent 4 }}
   name: {{ include "resource.policyException.name" . }}
-  namespace: giantswarm
```

Comment:
Why ?

Reply:
I mean this should have no effect

Reply:
something wrong with policy, so I try some stuffs.
But that change doesn't fix the issue.